addons/zeus/functions/fnc_ui_groupSide.sqf

```diff
@@ -128,7 +128,7 @@ private _fnc_onConfirm = {
     private _unit = effectiveCommander (attachedTo _logic);
     private _side = [west,east,independent,civilian] select (GETVAR(_display,newSide,GETVAR(_display,oldSide,0)));
 
-    [_unit, _side] call FUNC(moduleGroupSide);
+    [_unit, _side, _logic] call FUNC(moduleGroupSide);
     deleteVehicle _logic;
```

Comment:
I should probably remove this one so it fits every other module function, where the module gets deleted at the end of the moduleDERP.sqf function.